doc/man/8/mount.ceph.rst

```diff
@@ -9,8 +9,7 @@
 Synopsis
 ========
 
-| **mount.ceph** [*mon1_socket*\ ,\ *mon2_socket*\ ,...]:/[*subdir*] *dir* [
-  -o *options* ]
+| **mount.ceph** *name*@*fs_name*[.*fsid*]=/[*subdir*] *dir* [-o *options* ]
```

Comment:
Since you're dropping the legacy syntax here, I assume the plan is to have the mount helper convert new syntax to old if it encounters an old kernel?

Reply:
right -- mount.ceph falls back to `mon_ip:/path` in such cases.